test/units/testsuite-58.sh

```diff
@@ -154,13 +154,14 @@ device: $imgs/zzz
 unit: sectors
 first-lba: 2048
 last-lba: 2097118
-$imgs/zzz1 : start=        2048, size=      591856, type=933AC7E1-2EB4-4F13-B844-0E14E2AEF915, uuid=4980595D-D74A-483A-AA9E-9903879A0EE5, name=\"home-first\", attrs=\"GUID:59\"
-$imgs/zzz4 : start=     1777624, size=      131072, type=0657FD6D-A4AB-43C4-84E5-0933C84B4F4F, uuid=78C92DB8-3D2B-4823-B0DC-792B78F66F1E, name=\"swap\""
+$imgs/zzz1 : start=        2048, size=     1775576, type=933AC7E1-2EB4-4F13-B844-0E14E2AEF915, uuid=4980595D-D74A-483A-AA9E-9903879A0EE5, name=\"home-first\", attrs=\"GUID:59\"
+$imgs/zzz2 : start=     1777624, size=      131072, type=0657FD6D-A4AB-43C4-84E5-0933C84B4F4F, uuid=78C92DB8-3D2B-4823-B0DC-792B78F66F1E, name=\"swap\""
 
     runas testuser systemd-repart --definitions="$defs" \
                                   --dry-run=no \
                                   --seed="$seed" \
-                                  --exclude-partitions=root \
+                                  --empty=force \
+                                  --skip-partitions=root \
```

Comment:
can you skip a couple of parts instead of just one? to make sure we don't have weird ordering issues or such